src/oops/SOLID/singleResponsibilityPrinciple/before/EmployeeDao.java

```diff
@@ -0,0 +1,5 @@
+package oops.SOLID.singleResponsibilityPrinciple.before;
+
+public interface EmployeeDao {
```

Comment:
It is really hard to understand with the name that you have given to this interface.

Reply:
DAO or Data Access Object is an object/interface which is used to access data from database . To keep it simple, EmployeeDao provides an abstraction for the CRUD operations on Employee type of object. 
Reference: https://stackoverflow.com/questions/19154202/data-access-object-dao-in-java